plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```
@@ -88,6 +122,407 @@
+	 */
+	private function get_paypal_order_items( $order ) {
+		$items = array();
+
+		foreach ( $order->get_items() as $item ) {
+			$items[] = array(
+				'name'        => $item->get_name(),
+				'quantity'    => $item->get_quantity(),
+				'unit_amount' => array(
+					'currency_code' => $order->get_currency(),
+					// We want to use the subtotal (before discounts), as we are including the discount in the breakdown.
+					'value'         => $order->get_item_subtotal( $item, $include_tax = false, $rounding_enabled = false ),
```

PHPMD (2.15.0)

Avoid unused local variables such as '$include_tax'. (Unused Code Rules)

(UnusedLocalVariable)